mathics/builtin/optimization.py

```diff
@@ -16,6 +16,10 @@
 
 import sympy
 
+from mathics.core.symbols import Symbol
+
+SymbolConstant = Symbol("System`Constant")
```

Comment:
Maybe import this from `mathics.core.systemsymbols`?

Reply:
Here is a point I wanted to discuss with you: there are certain symbols that appears in a very specific context. Do we want to define them in systemsymbols, or just in the corresponding module? At the end, if we define the same SymbolSymbolName in different places, all of them are references to the same symbol. What is important is to avoid to call `Symbol("Name")` inside method.

Reply:
Done